app/helpers/blacklight/blacklight_helper_behavior.rb

```diff
@@ -79,7 +79,14 @@ def extra_body_classes
   # Render the search navbar
   # @return [String]
   def render_search_bar
-    search_bar_presenter.render
```

Comment:
I'm worried about the backwards-compatible implications of skipping the presenter here. I think downstream applications could have overridden that or the `catalog/search_form` partial.

Reply:
How would you feel about checking the class of the presenter, and dispatching to the subclass if it is not the default?

Reply:
Added a commit that falls back to the deprecated behavior if the presenter class is not the default, or if the partial does not appear to be the default partial.